core/domain/feedback_jobs_one_off.py

```diff
@@ -66,3 +67,33 @@ def map(item):
     @staticmethod
     def reduce(key, instance_ids):
         yield (key, instance_ids)
+
+
+class PopulateMessageCountOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for populating the message count field."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Returns a list of datastore class references to map over."""
+        return [feedback_models.GeneralFeedbackThreadModel]
+
+    @staticmethod
+    def map(item):
```

Comment:
can we rename item -> thread (as we know it will be a thread).

Reply:
Okay sure .